datastore/data_quality/management/commands/rewrite_quality_data.py

```diff
@@ -82,7 +88,7 @@ def handle(self, *args, **options):
                     }
                 )
 
-            with Pool(8) as process_pool:
+            with Pool(options.get("threads")) as process_pool:
```

Comment:
I think `options.threads` here? Pool can't take None and it's been initialised through through the default